borg/helpers.py

```diff
@@ -1089,3 +1091,31 @@ def log_multi(*msgs, level=logging.INFO):
         lines.extend(msg.splitlines())
     for line in lines:
         logger.log(level, line)
+
+
+class ErrorIgnoringTextIOWrapper(io.TextIOWrapper):
+    def read(self, n):
+        if not self.closed:
+            try:
+                return super(ErrorIgnoringTextIOWrapper, self).read(n)
+            except OSError as err:
+                if err.errno != errno.EPIPE:
+                    raise
```

Comment:
That doesn't exist for whatever reason, and I can't find any mention of it in the docs. I think it's just how the error prints out maybe?